source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -40,20 +40,20 @@ Alright, that's it! Enjoy the release!
 
 ../Frenck
 
-- [New entity: Select](#new-entity-select)
-- [Trigger conditions and trigger IDs](#trigger-conditions-and-trigger-ids)
-- [Script debugging](#script-debugging)
-- [Referencing other entities in triggers and conditions](#referencing-other-entities-in-triggers-and-conditions)
-- [Working with dates in templates](#working-with-dates-in-templates)
-- [Series version tags for Docker containers](#series-version-tags-for-docker-containers)
-- [Other noteworthy changes](#other-noteworthy-changes)
-- [New Integrations](#new-integrations)
-- [New Platforms](#new-platforms)
-- [Integrations now available to set up from the UI](#integrations-now-available-to-set-up-from-the-ui)
-- [Release 2021.7.1 - July 8](#release-202171---july-8)
-- [If you need help...](#if-you-need-help)
-- [Breaking Changes](#breaking-changes)
-- [All changes](#all-changes)
+- [New entity: Select]({{page.url}}#new-entity-select)
```

Comment:
Meh, this is generated... :(
While this fixes a single case, this has been broken and will break again.

Not saying this is a bad solution, I'm just wondering if there is something we can do to prevent it from happening.

Reply:
How is this generated?
Looks hardcoded in this file

Reply:
Its created and updated by vscode.

Reply:
that sucks